.github/workflows/release.yml

```diff
@@ -165,7 +165,7 @@ jobs:
 
       - name: Upload to S3
         if: ${{ github.event.pull_request.number }}
-        uses: shallwefootball/s3-upload-action@master
+        uses: shallwefootball/upload-s3-action@0d261a6f15b3b2e209dfebdecace4b100c04f95b
```

Comment:
can we use the github action that we use in decentraland instead of this one ? (also the name its quite akward)
I think we use @dcl/cdn-uploader or something like that

Reply:
yes! was just doing the change